tests/integration/api_v3/constants/common.js

```diff
@@ -0,0 +1,26 @@
+/*
+ * LiskHQ/lisk-service
+ * Copyright © 2022 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ *
+ */
+const CHAIN_ID_PREFIX_NETWORK_MAP = Object.freeze({
```

Comment:
If we already have this defined anywhere else within the codebase (probably from within the `app-registry`!?), could we please refer to config that and remove this instead? Please keep in mind that we're removing the entry from within the `blockchain-indexer` in a separate PR.

Reply:
Applied.